mocha-reporter-config-plugins.json

```diff
@@ -0,0 +1,6 @@
+{
+  "reporterEnabled": "spec, mocha-junit-reporter",
+  "mochaJunitReporterReporterOptions": {
+    "mochaFile": "./plugins-test-results/mocha/test-results.xml"
```

Comment:
Why not just keep the default?

Reply:
this is purely for separating the test results in CircleCI. If we don't want to use that, I can sue the same

Reply:
Not sure I understand. Each test runs in its own job, so aren't they already separated?

Reply:
And I guess locally we have the opposite problems, where running multiple plugins back to back would override the file. I would say to keep the default since this will only be useful in CircleCI anyway and jobs are already split.

Reply:
I understood that even if you run in different jobs, the "insights" would gather those results. 

https://circleci.com/docs/2.0/configuration-reference/#storetestresults
> Note: Please write your tests to subdirectories of your store_test_results path, ideally named to match the names of your particular test suites, in order for CircleCI to correctly infer the names of your reports. If you do not write your reports to subdirectories, you will see reports in your “Test Summary” section such as Your build ran 71 tests in unknown, instead of, for example, Your build ran 71 tests in rspec.

I understood this as "keep results in different folders". But in this case, even if they merge, it is fine. So yeah, I'll remove this file

Reply:
We'll see if we need to adjust anything after this is merged, but if we do I expect that we'll need a way to identify individual jobs, which will be more complicated.